builtin/providers/rabbitmq/resource_binding.go

```diff
@@ -0,0 +1,160 @@
+package rabbitmq
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/michaelklishin/rabbit-hole"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceBinding() *schema.Resource {
+	return &schema.Resource{
+		Create: CreateBinding,
+		Update: nil,
```

Comment:
If all are ForceNew then we don't even need to include the Update part of the schema